GrandCentralBoard/Widgets/Harvest/UIColor+BillingColor.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIColor+BillingColor.swift
```

Comment:
In this project, I see more extensions without the oldschool `+` suffix. Let's keep it consistent. Also, consider moving this file to `Extensions` group, like Bonus widget does.